test/types/string/psahabu/perf/arguments.chpl

```diff
@@ -0,0 +1,45 @@
+use Time;
+
+config const n = 1000000;
+config const timing = true;
+
+// passing
+var pass = "passing a string";
+var acc = 0;
+var tPassing: Timer;
+if timing then tPassing.start();
+for i in 1..n {
+  acc += receive(pass);
+}
+if timing then tPassing.stop();
+
+// returning
+var tReturning: Timer;
+if timing then tReturning.start();
+for i in 1..n {
+  var s = send(i);
```

Comment:
Same here, so that 's' doesn't get eliminated since it's never used. I might be too paranoid here...